lib/RGBot.js

```diff
@@ -434,6 +465,8 @@ class RGBot extends EventEmitter {
      * Minecraft normally runs at 20 ticks per second, with an in-game day lasting 24,0000 ticks (20 minutes).
      * This is similar to the standard JavaScript setTimeout function but runs on the physics timer of the Bot specifically.
      * This is useful for waiting on the server to update a Block or spawn drops when you break a Block.
+     * If you'd like to wait for a specific amount of time, see the `waitForMilliseconds` function.
```

Comment:
This is more of a personal preference, but I tend to avoid referring to the user as "you". We should try to keep the tone consistent throughout the documentation, in which case I'd expect something more along the lines of "see `waitForMilliseconds` for time-based rather than tick-based waits."

Reply:
Fixed!